docs/book/src/user/quick-start.md

```diff
@@ -731,7 +731,19 @@ spec:
 {{#/tab }}
 {{#/tabs }}
 
-After the controlplane is up and running, let's retrieve the [target cluster] Kubeconfig:
+To verify the control plane is up, check if the control plane machine
+has a ProviderID.
+```
+kubectl get machines --selector cluster.x-k8s.io/control-plane
+```
+
+If the control plane is deployed via [KubeadmControlPlane
+controller][control-plane-controller] ensure the control plane is ready.
```

Comment:
I wouldn't mention "controller" - users won't necessarily know what a controller is. Should we also try to be a bit more generic? Something like:

If the control plane is deployed using a control plane provider, such as KubeadmControlPlane, ensure the control plane is ready.

(then have them look at cluster.status.controlPlaneInitialized and cluster.status.controlPlaneReady)